ExampleMod/Content/NPCs/ExamplePerson.cs

```diff
@@ -50,15 +52,18 @@ public class ExamplePerson : ModNPC
 
 			// Set Example Person's biome and neighbor preferences with the NPCHappiness hook. You can add happiness text and remarks with localization (See an example in ExampleMod/Localization/en-US.lang).
 
-			// Biomes
-			NPC.Happiness.LikeBiome(PrimaryBiomeID.Forest); // Example Person prefers the forest.
-			NPC.Happiness.DislikeBiome(PrimaryBiomeID.Snow); // Example Person dislikes the snow.
-			NPC.Happiness.LoveBiome(ModContent.GetInstance<ExampleSurfaceBiome>().Type); // Example Person likes the Example Surface Biome
-			// NPCs
-			NPC.Happiness.HateNPC(NPCID.Demolitionist); // Hates living near the demolitionist.
-			NPC.Happiness.DislikeNPC(NPCID.Merchant); // Dislikes living near the merchant.
-			NPC.Happiness.LikeNPC(NPCID.Guide); // Likes living near the guide.
-			NPC.Happiness.LoveNPC(NPCID.Dryad); // Loves living near the dryad.
+			NPC.Happiness
+				// Biomes
```

Comment:
I have concerns with this formatting. It may confuse new modders. The comments between the initial field/property and the builder method make this a little less clear than with recipes.